wolftpm/tpm2_types.h

```diff
@@ -208,10 +212,24 @@ typedef int64_t  INT64;
 
 #ifndef WOLFTPM_CUSTOM_TYPES
     #include <stdlib.h>
+
+    #ifndef XHTONS
+        /* WOLFCRYPT_ONLY means no wolfio and no arpa/inet.h */
+        #ifdef WOLFCRYPT_ONLY
```

Comment:
I'm not sure assuming wolfCrypt only indicates this. Maybe just gate on `XHTONS` only?

Reply:
I'm not sure, the real htons is in arpa/inet.h and anytime that gets included in wolfBoot through lib/ it breaks and says cannot find since wolfBoot isn't supposed to use OS level libraries. wolfBoot defines WOLFCRYPT_ONLY so I gated on that since otherwise wolfTPM should get the real htons